staging/src/k8s.io/apimachinery/pkg/runtime/serializer/cbor/internal/modes/encode.go

```diff
@@ -75,6 +75,14 @@ var Encode cbor.EncMode = func() cbor.EncMode {
 		// Encode struct field names to the byte string type rather than the text string
 		// type.
 		FieldName: cbor.FieldNameToByteString,
+
+		// Marshal []byte to CBOR byte string enclosed in tag 22 (expected later base64
+		// encoding, https://www.rfc-editor.org/rfc/rfc8949.html#section-3.4.5.2), to
+		// interoperate with the existing JSON behavior. This indicates to the decoder that,
+		// when decoding into a string (or unstructured), the resulting value should be the
+		// base64 encoding of the original bytes. No base64 encoding or decoding needs to be
+		// performed for []byte-to-CBOR-to-[]byte roundtrips.
+		ByteSliceLaterFormat: cbor.ByteSliceLaterFormatBase64,
```

Comment:
is this encoder used in the staging/src/k8s.io/apimachinery/pkg/runtime/serializer/cbor/internal/modes/appendixa_test.go test?

Reply:
Yes.